src/modal/modal.component.ts

```diff
@@ -24,60 +24,60 @@ import { cycleTabs, getFocusElementList } from "./../common/tab.service";
 /**
  * Component to create modals for presenting content.
  *
- * Using a modal in your application requires `ibm-modal-placeholder` which would generally be
+ * Using a modal in your application requires `ibm-placeholder` which would generally be
  * placed near the end of your app component template (app.component.ts or app.component.html) as:
  *
- * ```html
- * <ibm-modal-placeholder></ibm-modal-placeholder>
- * ```
+```html
+<ibm-modal-placeholder></ibm-modal-placeholder>
+```
  *
  * A more complete example for `Modal` is given as follows:
  *
  * Example modal definition:
  *
- * ```typescript
- * \@Modal()
- * \@Component({
- * 	selector: "app-sample-modal",
- * 	template: `
- *		<ibm-modal size="xl">
- * 			<ibm-modal-header (closeSelect)="closeModal()">Header text</ibm-modal-header>
- * 			<section class="modal-body">
- * 			<h1>Sample modal works.</h1>
- * 			<button class="btn--icon-link" nPopover="Hello there" title="Popover title" placement="right" appendInline="true">
- * 				<ibm-icon icon="info" size="sm"></ibm-icon>
- * 			</button>
- * 			{{modalText}}
- * 			</section>
- * 			<ibm-modal-footer><button ibmButton="primary" (click)="closeModal()">Close</button></ibm-modal-footer>
- * 		</ibm-modal>`,
- * 	styleUrls: ["./sample-modal.component.scss"]
- * })
- * export class SampleModal extends BaseModal {
- * 	modalText: string;
- * 	constructor(protected injector: Injector) {
- * 		super();
- * 		this.modalText = this.injector.get("modalText");
- * 	}
- * }
- * ```
+```typescript
+@Component({
```

Comment:
Docs had `\` in before `@` because of some compodoc display issue, that's fixed now?

Reply:
Seems to be ... everything displays correctly in compodoc now (including the `\`, so I removed it)